eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -56,6 +56,10 @@ jobs:
       - script: sudo xcode-select --switch /Applications/Xcode_$(XcodeVersion).app
         displayName: 'Use Xcode $(XcodeVersion)'
 
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
+        parameters:
+          Directory: sdk/${{ parameters.ServiceDirectory }}
```

Comment:
@mitchdenny @bsiegel correct me if I'm wrong but I don't think there are service directories in the ios repo. I thought there is only one large client build. In which case we should just run the link checker on the entire repo.

Reply:
We do have service directories, but currently we have a single pipeline that builds all libraries. It can easily be swapped between the two options by simply changing the scheme name (from "all" to the name of the service directory) when calling xcodebuild.

That said, yes I'd support running the link checker on the entire repo during each build.

Reply:
Will check the entire repo here in new PR.